<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>502</width>
    <height>586</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>504</width>
    <height>586</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>502</width>
    <height>586</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Esp8266Debugger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>wifi_PNG12.png</normaloff>wifi_PNG12.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>458</width>
             <height>257</height>
            </size>
           </property>
           <property name="title">
            <string>Serial Port Setup</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(157, 216, 255);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Panel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>  Port  </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBox_1"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(157, 216, 255);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Panel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string>  Baud  </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBox_2"/>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="pushButton_1">
                <property name="text">
                 <string>Connect</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Disconnect</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="P_REFRESH">
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Messages</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QTextBrowser" name="textBrowser_1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>209</width>
                   <height>146</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>209</width>
                   <height>146</height>
                  </size>
                 </property>
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:5pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Basic Commands</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="L_ATGMR">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+GMR</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="L_AT">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="L_ATRST">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+RST</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="L_ATCWLAP">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWLAP</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QPushButton" name="P_ATCWJAPE">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="L_ATCWJAPQ">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWJAP?</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="B_ATCWJAP_1"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="L_ATCWJAPE">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWJAP=</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="B_ATCWJAP_2"/>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="P_ATCWJAPQ">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="P_ATCWLAP">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="P_ATRST">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="P_ATGMR">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="P_AT">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Advanced Commands</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTextBrowser" name="textBrowser_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="1" column="2">
             <widget class="QLineEdit" name="B_ATCWSAP_2"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="B_ATCIPSEND_1"/>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="B_ATCIPSTART_2"/>
            </item>
            <item row="5" column="4">
             <widget class="QPushButton" name="P_ATCIPSTART">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="P_ATCWSAPE">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLineEdit" name="B_ATCIPSTART_3"/>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="P_ATCWSAPQ">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="L_ATCWSAPE">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWSAP=</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="P_ATCIPSTATUS">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="L_ATCWSAPQ">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWSAP?</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="L_ATCIPSTART">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CIPSTART</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="L_ATCIPSTATUS">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CIPSTATUS</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="L_ATCIPSEND">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CIPSEND</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="L_ATCIFSR">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CIFSR</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="B_ATCWSAP_1"/>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="P_ATCIFSR">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="P_ATCWQAP">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QPushButton" name="P_ATCIPSEND">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="L_ATCWQAP">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWQAP</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="B_ATCIPSTART_1"/>
            </item>
            <item row="7" column="1">
             <widget class="QPushButton" name="P_ATCWMODEQ">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="L_ATCWMODEQ">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWMODE?</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(157, 216, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>AT+CWMODE=</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QPushButton" name="P_ATCWMODEE">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="B_ATCWMODE_1"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_1</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>P_REFRESH</tabstop>
  <tabstop>textBrowser_1</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>textBrowser_2</tabstop>
  <tabstop>P_ATCWSAPQ</tabstop>
  <tabstop>B_ATCWSAP_1</tabstop>
  <tabstop>B_ATCWSAP_2</tabstop>
  <tabstop>P_ATCWSAPE</tabstop>
  <tabstop>P_ATCWQAP</tabstop>
  <tabstop>P_ATCIFSR</tabstop>
  <tabstop>P_ATCIPSTATUS</tabstop>
  <tabstop>B_ATCIPSTART_1</tabstop>
  <tabstop>B_ATCIPSTART_2</tabstop>
  <tabstop>B_ATCIPSTART_3</tabstop>
  <tabstop>P_ATCIPSTART</tabstop>
  <tabstop>B_ATCIPSEND_1</tabstop>
  <tabstop>P_ATCIPSEND</tabstop>
  <tabstop>P_ATCWMODEQ</tabstop>
  <tabstop>B_ATCWMODE_1</tabstop>
  <tabstop>P_ATCWMODEE</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
